src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
Okay so I don't know why we cannot just take the all the packages from dependencies in the `package.json` https://github.com/dojo/cli-build/blob/master/package.json#L49

If there are any missing or in the incorrect section (devDep) we should fix that rather than maintaining an external list (without versions mind you) as a work around.